src/js/monkey.js

```diff
@@ -10,7 +10,7 @@ window.Monkey = module.exports = (function () {
    */
   function Monkey(monkeyContainer, options) {
     var $monkeyContainer = $(monkeyContainer);
-    var pickerLocales = ['en-GB', 'en-US', 'fr'];
```

Comment:
Hmm, I find it odd that we'd hardcode this here when this data is already available in `lmn-assets`... It doesn't feel very DRY.

I think whatever invokes `monkey` on Eagle should control this...